src/Configuration.ts

```diff
@@ -0,0 +1,32 @@
+import * as joi from '@hapi/joi';
+import 'joi-extract-type';
+
+import { probeSchema } from './probes';
+
+export const defaultProbeEvaluateInterval = 60000;
+export const defaultStabilizeAfterNSuccesses = 2;
+
+const topSchema = joi.object({
+    probes: joi.array().items(probeSchema).optional(),
+    nominalEvaluateIntervalMs: joi.number().default(defaultProbeEvaluateInterval).optional(),
+    trippedEvaluateIntervalMs: joi.number().default(defaultProbeEvaluateInterval).optional(),
+    stabilizingEvaluateIntervalMs: joi.number().default(defaultProbeEvaluateInterval).optional(),
+    stabilizeAfterNSuccesses: joi.number().default(defaultStabilizeAfterNSuccesses).optional(),
```

Comment:
Might want to add `.positive()` to these
```suggestion
    nominalEvaluateIntervalMs: joi.number().positive().default(defaultProbeEvaluateInterval).optional(),
    trippedEvaluateIntervalMs: joi.number().positive().default(defaultProbeEvaluateInterval).optional(),
    stabilizingEvaluateIntervalMs: joi.number().positive().default(defaultProbeEvaluateInterval).optional(),
    stabilizeAfterNSuccesses: joi.number().positive().default(defaultStabilizeAfterNSuccesses).optional(),
```